modules/client-configuration/pages/snippets/appstream_admon.adoc

```diff
@@ -5,6 +5,8 @@ This results in the {productname} {webui} showing incorrect package information.
 You cannot perform package operations such as installing or upgrading directly from modular repositories using the {webui} or API.
 
 You can use the AppStream filter with content lifecycle management (CLM) to transform modular repositories into regular repositories.
+Make sure to include ``python:3.6`` using an AppStream filter.
```

Comment:
... if you want to use `spacecmd` on the clients.

In most cases people do not need spacecmd and if this is the case, they can also use other python versions.